packages/aws-cdk/bin/cdk.ts

```diff
@@ -31,7 +28,7 @@ async function parseCommandLineArguments() {
   return yargs
     .env('CDK')
     .usage('Usage: cdk -a <cdk-app> COMMAND')
-    .option('app', { type: 'string', alias: 'a', desc: 'REQUIRED: Command-line for executing your CDK app (e.g. "node bin/my-app.js")', requiresArg: true })
+    .option('app', { type: 'string', alias: 'a', desc: 'REQUIRED: command-line for executing your app or a cloud assembly directory (e.g. "node bin/my-app.js")', requiresArg: true })
```

Comment:
The cloud-assembly form is only valid for `deploy`, not for `synthesize`. I feel the CLI documentation does a poor job at explaining this. Maybe we should move this to be under all commands that allow it, with a correct text for each.

Reply:
It is valid for synth as well actually. You could use it to extract a template for a stack like this:

```console
$ cdk synth -a <cloud-assembly-dir> <stack-name>
```